frontend/app/src/App.test.tsx

```diff
@@ -774,6 +775,29 @@ describe("App.handleNewSession", () => {
       // @ts-expect-error
       window.history.pushState.mockClear()
     })
+
+    it("logs a warning when hostConfig.disableSetPageMetadata is true", async () => {
```

Comment:
nit: Can you add the title to this test as well?

Reply:
Can you clarify what you mean? Do you want me to add something to this title of the test? Or do a check on title?

The current implementation looks like this in `App.handleNewSession`:
```
if (this.state.hostConfig.disableSetPageMetadata) {
      logWarning(
        "Setting the page metadata (title & favicon) is disabled by security policy of the host."
      )
    } else {
      // Set the title and favicon to their default values
      document.title = `${newPageName} · Streamlit`
      handleFavicon(
        `${process.env.PUBLIC_URL}/favicon.png`,
        this.hostCommunicationMgr.sendMessageToHost,
        this.endpoints
      )
    }
```

Reply:
ahh, I see. There is a combined warning. Then we can leave it as is 👍